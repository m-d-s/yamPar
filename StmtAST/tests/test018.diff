1,3d0
< Block
<   Empty
<   Empty
