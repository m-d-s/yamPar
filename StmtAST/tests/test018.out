Call:   Top
  Call:   Stmt
    Call:   Block
      Consumed token: <"{" at line 1 column 1>
      Call:   Stmts
        Call:   Stmt
          Call:   Empty
            Consumed token: <";" at line 1 column 2>
          Return: Empty
        Return: Stmt
        Call:   Stmts
          Call:   Stmt
            Call:   Empty
              Consumed token: <";" at line 1 column 3>
            Return: Empty
          Return: Stmt
          Call:   Stmts
          Return: Stmts
        Return: Stmts
      Return: Stmts
      Consumed token: <"}" at line 1 column 4>
    Return: Block
  Return: Stmt
  Consumed token: <<EOF> at line 1 column 51>
Return: Top
Block
  Empty
  Empty
